### PR TITLE
UI updates.

### PR DESCRIPTION
**Pull request by CarbonGrid**
# STATUS 
1. Standardises commands in AutoComplete.
2. Implements Check For Update and About Us MenuItems.
3. Simplifies and enables horizontal scrolling for long strings in cells.
4. Removes addition eager evalution call in MainWindow.

### CODE REVIEW

File: src/main/java/seedu/address/ui/autocomplete/AutoCompleter.java [hunk 1]
```diff
@@ -42,14 +42,14 @@
...
     private static final Map<String, Set<String>> SUPPORTED_ARGUMENTS = Map.ofEntries(
-            Map.entry("addappt", Set.of("-id", "-rec", "-num", "-start")),
-            Map.entry("addshift", Set.of("-id", "-rec", "-num", "-start")),
-            Map.entry("changeappt", Set.of("-start")),
-            Map.entry("changeshift", Set.of("-start")),
-            Map.entry("edit", Set.of("-id", "-name", "-phone", "-address", "-tag", "-email")),
+            Map.entry("editappt", Set.of("-start")),
+            Map.entry("editshift", Set.of("-start")),
+            Map.entry("editdoctor", Set.of("-id", "-name", "-phone", "-address", "-email")),
+            Map.entry("editpatient", Set.of("-id", "-name", "-phone", "-address", "-tag", "-email")),
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/ui/autocomplete/AutoCompleter.java]: can add the "-entry" to these 4 commands

File: src/main/java/seedu/address/ui/autocomplete/AutoCompleter.java [hunk 2]
```diff
@@ -42,14 +42,14 @@
  */
 public class AutoCompleter {
     private static final Map<String, Set<String>> SUPPORTED_ARGUMENTS = Map.ofEntries(
-            Map.entry("addappt", Set.of("-id", "-rec", "-num", "-start")),
-            Map.entry("addshift", Set.of("-id", "-rec", "-num", "-start")),
-            Map.entry("changeappt", Set.of("-start")),
-            Map.entry("changeshift", Set.of("-start")),
-            Map.entry("edit", Set.of("-id", "-name", "-phone", "-address", "-tag", "-email")),
+            Map.entry("editappt", Set.of("-start")),
+            Map.entry("editshift", Set.of("-start")),
```
Comment (SakuraBlossom) [inline src/main/java/seedu/address/ui/autocomplete/AutoCompleter.java]: Can add the "-end" flag to these to commands

Comment (CarbonGrid) [reply src/main/java/seedu/address/ui/autocomplete/AutoCompleter.java]: Done.

File: src/main/java/seedu/address/ui/autocomplete/AutoCompleter.java (hunk 1 above)
Comment (CarbonGrid) [reply src/main/java/seedu/address/ui/autocomplete/AutoCompleter.java]: Done.

**Pull request merged**